.babelrc

```diff
@@ -1,7 +1,7 @@
 {
   "presets": [
     "react",
-    ["env", {"targets": {"browsers": ["firefox >= 54"]}}]
+    ["env", {"targets": {"browsers": ["firefox >= 56"]}}]
```

Comment:
56 or 57?

Reply:
56, since Babel isn't happy with 57 (they haven't updated their env presets, at least not when this was committed).